src/main/scala/beam/router/model/EmbodiedBeamTrip.scala

```diff
@@ -14,6 +14,7 @@ import beam.router.Modes.BeamMode.{
   WALK,
```

Comment:
What are the changes in this file? Is it just re-arranged? If yes, please, undo it otherwise investigation can be a bit harder (it will require some brain power to understand that there were no actual code change)

Reply:
the method `determineTripMode` was moved from the class to the companion